Systems/c172p-engine.xml

```diff
@@ -206,13 +206,13 @@
         </switch>
 
         <fcs_function name="auto-engine-mixture">
-            <function>
+            <function> <!-- mixture to 1 at sea level in autostart, state autostarts and tutorials, default mixture setting is .95 -->
                 <sum>
                     <value>0</value>
                     <table>
                         <independentVar>/position/altitude-ft</independentVar>
                         <tableData>
-                            0       0.95
+                            0       1.00
```

Comment:
@wlbragg Doesn't this already starts lowering the mixture above 0 ft? (linear interpolation between 0 and 13_000 ft). Perhaps insert a row `x 1.00` where x is an altitude where 100 % can still be safely used.

Reply:
Makes sense, I'll run some tests and see where it should start decreasing.

Reply:
I also had this thought.

The POH-180hp/552SP says:
(p. 4-16, PDF 100/384)

> ENROUTE CLIMB
> 3. Mixture -- RICH (above 3000 feet, LEAN to obtain maximum RPM).

Reply:
Thanks @dany93 that'll give us some kind of a hint or target to shoot for.